pkgs/development/python-modules/sjcl/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "sjcl";
+  version = "0.2.1";
+
+  format = "pyproject";
+  disabled = pythonOlder "3.5";
```

Comment:
The trove classifiers say 2.7, 3.5 and 3.6 are supported, so I think we might as well drop this statement entirely.

Reply:
removed with [e119b23](https://github.com/NixOS/nixpkgs/pull/167951/commits/e119b230877703a025cd50aa9b317f686ab46dd6)